pasta/base/ast_utils.py

```diff
@@ -117,15 +117,16 @@ def space_between(from_loc, to_loc, line, lines):
 def setup_props(node):
   if not hasattr(node, PASTA_DICT):
     try:
-      setattr(node, PASTA_DICT, collections.defaultdict(lambda: ''))
+      setattr(node, PASTA_DICT, collections.defaultdict(lambda: None))
     except AttributeError:
       pass
 
 
 def prop(node, name):
-  if hasattr(node, PASTA_DICT):
+  try:
```

Comment:
i kind of prefer the other way but maybe thats just me

Reply:
This AIUI is more pythonic, because `getattr` performs the same check as `hasattr` and raises an exception on failure, so the `hasattr` check is redundant. Doesn't make a big difference either way.